src/main/java/com/iexec/sms/authorization/AuthorizationService.java

```diff
@@ -132,7 +130,7 @@ public String getChallengeForSetWeb3Secret(String secretAddress,
                 Hash.sha3String(secretValue));
     }
 
-    public String getChallengeForSetAppRuntimeSecret(String appAddress,
+    public String getChallengeForSetAppComputeSecret(String appAddress,
```

Comment:
```suggestion
    public String getChallengeForSetAppDeveloperAppComputeSecret(String appAddress,
```

Reply:
Rest of the first story, thanks for pointing it out!